etc/config.xml

```diff
@@ -55,5 +55,77 @@
                 <stockprice>0 8 * * *</stockprice>
             </cron>
         </sinchimport>
+        <csp>
+            <mode>
+                <storefront>
+                    <report_only>1</report_only>
+                </storefront>
+                <admin>
+                    <report_only>1</report_only>
+                </admin>
+            </mode>
+            <policies>
+                <storefront>
+                    <objects>
+                        <policy_id>object-src</policy_id>
+                        <self>0</self>
+                        <inline>0</inline>
+                        <eval>0</eval>
+                        <dynamic>0</dynamic>
+                    </objects>
+                    <base>
+                        <policy_id>base-uri</policy_id>
+                        <self>0</self>
+                        <inline>0</inline>
+                        <eval>0</eval>
+                        <dynamic>0</dynamic>
+                    </base>
+                    <frame-ancestors>
+                        <policy_id>frame-ancestors</policy_id>
+                        <self>1</self>
+                        <inline>0</inline>
+                        <eval>0</eval>
+                        <dynamic>0</dynamic>
+                    </frame-ancestors>
+                    <images>
+                        <policy_id>img-src</policy_id>
+                        <values>
+                            <value id="media.sitc" type="host">https://media.stockinthechannel.com</value>
+                        </values>
+                    </images>
+
+
+                </storefront>
+                <admin>
+                    <objects>
+                        <policy_id>object-src</policy_id>
+                        <self>0</self>
+                        <inline>0</inline>
+                        <eval>0</eval>
+                        <dynamic>0</dynamic>
+                    </objects>
+                    <base>
+                        <policy_id>base-uri</policy_id>
+                        <self>0</self>
+                        <inline>0</inline>
+                        <eval>0</eval>
+                        <dynamic>0</dynamic>
+                    </base>
+                    <frame-ancestors>
+                        <policy_id>frame-ancestors</policy_id>
+                        <self>1</self>
+                        <inline>0</inline>
+                        <eval>0</eval>
+                        <dynamic>0</dynamic>
+                    </frame-ancestors>
+                    <images>
+                        <policy_id>img-src</policy_id>
+                        <values>
+                            <value id="media.sitc" type="host">https://media.stockinthechannel.com</value>
```

Comment:
Same as above, this doesn't do anything. Host rules are read from csp_whitelist.xml as documented [here](https://developer.adobe.com/commerce/php/development/security/content-security-policies/#add-a-domain-to-the-whitelist)